rewrite-gradle/src/main/java/org/openrewrite/gradle/UpgradeDependencyVersion.java

```diff
@@ -208,18 +211,15 @@ public J visitMethodInvocation(J.MethodInvocation method, ExecutionContext ctx)
                         if(acc.gaToNewVersion.containsKey(ga)) {
                             return m;
                         }
-                        String oldVersion = findCurrentResolvedVersion(groupId, artifactId, m.getSimpleName(), gradleProject);
-                        if(oldVersion == null) {
-                            return m;
-                        }
                         try {
-                            String newVersion = findNewerProjectDependencyVersion(groupId, artifactId, oldVersion, gradleProject, ctx);
-                            if(oldVersion.equals(newVersion)) {
-                                return m;
-                            }
+                            String resolvedVersion = "classpath".equals(m.getSimpleName()) ?
```

Comment:
Previously, we weren't taking into account properties being used in the `buildscript` block.